drs_filer/ga4gh/drs/server.py

```diff
@@ -0,0 +1,57 @@
+import logging
+
+from flask import (current_app)
+from flask.globals import request
+
+from drs_filer.ga4gh.drs.endpoints import registerNewObject
+
+from werkzeug.exceptions import NotFound
+
+logger = logging.getLogger(__name__)
+
+
+def RegisterObject():
+    db_collection = (
+        current_app.config['FOCA'].db.dbs['drsStore'].
+        collections['objects'].client
+    )
+    response = registerNewObject.registerNewObject(
+        db_collection,
+        request.json
+    )
+    return response
+
+
+def GetObject(object_id):
+    db_collection = (
+        current_app.config['FOCA'].db.dbs['drsStore'].
+        collections['objects'].client
+    )
+
+    obj = db_collection.find_one_or_404({"id": object_id})
+    obj.pop("_id")
```

Comment:
Probably clearer to use either `del` or `.pop()` in the different endpoints, not both.